pkg/k8s/watchers/pod.go

```diff
@@ -719,6 +719,11 @@ func (k *K8sWatcher) deleteHostPortMapping(pod *slim_corev1.Pod, podIPs []string
 	}
 
 	for _, dpSvc := range svcs {
+		svcs := k.svcManager.GetDeepCopyServices()
+		svc := findServiceByL3n4Addr(dpSvc.Frontend.L3n4Addr, svcs)
+		if svc != nil && !deepEqualBackends(svc.Backends, dpSvc.Backends) {
```

Comment:
makes sense; can you add a comment for what this does?

Reply:
will do